README.md

```diff
@@ -1,3 +1,9 @@
 # Penultima Ideas
 
 Penultima ideas is the penultimate collection of penultima ideas.
+
+## Pawns
+
+### :poop: Not  pawn
```

Comment:
```suggestion
### :poop: Not a pawn
```

Reply:
A very good point, might have made that myself if I was sitting next to you and pointed the exact thing out.

Reply:
Level 4 headers for single pieces pls

Reply:
nevermind, this isn't the main.md